lib/app-tray.js

```diff
@@ -45,10 +42,20 @@ exports.init = function (gme) {
     type: 'separator'
   }));
   trayMenu.append(new MenuItem({
+    label: 'Show / Hide Google Music Electron',
+    click: gme.onTrayClick
+  }));
```

Comment:
Once the tray icon is patched on OS X (so it used the logo), then it should be patched since there is a linkage between the iconography. But that is a totally valid point for now